libs/facilities/frontend/view/src/lib/components/issues/issues.page.ts

```diff
@@ -4,19 +4,14 @@ import { RouterLink } from '@angular/router';
 import { IxModule } from '@siemens/ix-angular';
 
 import { XdBrowsePage } from '../browse';
-import { facilities } from '../facility.mocks/const';
 
 @Component({
 	selector: 'lib-issues',
 	standalone: true,
-	imports: [ CommonModule, IxModule, RouterLink, XdBrowsePage ],
+	imports: [CommonModule, IxModule, RouterLink, XdBrowsePage],
 	templateUrl: './issues.page.html',
 	styleUrl: './issues.page.scss',
 	encapsulation: ViewEncapsulation.None,
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class XdIssuesPage {
-	protected readonly _issueFacilities = facilities.filter(
-		(facility) => facility.variant != 'success',
-	);
-}
+export class XdIssuesPage {}
```

Comment:
Right now the facilities with issues are not filtered out at all. Should this component be deleted entirely and the filtering should be done in the browse page? Or how would you go about filtering out the ones with issues?

Reply:
I would delete the issues page and just handle it with the browse page, since the logic to everything is just the same, the only difference is that the issues page filters by default. 


I could not stay the logic as it is implemented since it would entirely blow up ticket size. `I add a comment again`

Reply:
I agree that it would blow up the ticket size I just asked because I was not sure how it should be done

Reply:
Lets talk about that tomorrow